### PR TITLE
:vpc configuration for lambda

### PR DESCRIPTION
**Pull request by zrzka**
Tested scenarions ...
# Create function without vpc
- `--vpc-config` not passed
# Create function with vpc
- `--vpc-config` passed
# Update function
- `update-function-configuration` called only if it differs (VPC & rest)
- `--vpc-config` not passed to `update-function-code`

### CODE REVIEW

**Comment by moea** [comment]
Thanks!  Will take a look and give it a run.

File: plugin/src/leiningen/cljs_lambda/aws.clj [hunk 1]
```diff
@@ -109,13 +143,28 @@
       255 nil
       0   (json/parse-string out csk/->kebab-case-keyword))))
 
+(defn normalize-config [config]
+  (as-> config x
+    (assoc-in x [:vpc :subnets] (sort (get-in x [:vpc :subnets])))
+    (assoc-in x [:vpc :security-groups] (sort (get-in x [:vpc :security-groups])))))
```
Comment (moea) [inline plugin/src/leiningen/cljs_lambda/aws.clj]: Would this be easier to follow as:

``` clojure
(-> config
  (update-in [:vpc :subnets] sort)
  (update-in [:vpc :security-groups] sort))
```

Comment (zrzka) [reply plugin/src/leiningen/cljs_lambda/aws.clj]: Thanks, pushed ;-)

**Comment by moea** [comment]
Looks good, will merge and deploy a snapshot.

**Comment by moea** [comment]
Done

**Pull request merged**